fuse/dl/lightning/pl_funcs.py

```diff
@@ -240,10 +251,15 @@ def epoch_end_compute_and_log_losses(
         for key in elem:
             if key not in losses:
                 losses[key] = []
-            if isinstance(elem[key], torch.Tensor):
-                losses(key).extend(elem[key].detach().cpu().tolist())
+            value = elem[key]
+            if isinstance(value, torch.Tensor):
+                value = value.detach()
+                if value.dtype == torch.bfloat16:
+                    value = value.to(torch.float)
+                value = value.cpu()
+                losses(key).extend(value.tolist())
```

Comment:
Weird, shouldn't it be with `[ ]`? 
i.e:
```python
losses[key].extend(value.tolist())
```
Otherwise we should get an error:
```
TypeError: 'dict' object is not callable
```

WDYT?